doc/source/whats_new.rst

```diff
@@ -31,6 +31,9 @@ Changelog
 
     - Add new object :class:`mne.decoding.TimeDecoding` for decoding sensors' evoked response across time by `Jean-Remi King`_
 
+    - Add support to append new channels from a list of Raw objects by `Chris Holdgraf`_
```

Comment:
Planning to do the rest of tests tomorrow... Should it be one function that
tests all the class types, or a separate "test_channel_update" test for
each type?
On Jul 16, 2015 10:58 PM, "Denis A. Engemann" <notifications@github.com>
wrote:

> In doc/source/whats_new.rst
> <https://github.com/mne-tools/mne-python/pull/2302#discussion_r34837076>:
>
> > @@ -31,6 +31,9 @@ Changelog
> >
> >      - Add new object :class:`mne.decoding.TimeDecoding` for decoding sensors' evoked response across time by `Jean-Remi King`_
> >
> > +    - Add support to append new channels from a list of Raw objects by `Chris Holdgraf`_
>
> Good point, let's test it before merging it.
> … <#14e98a9363755f71_>
>  On 16 Jul 2015, at 22:49, Alexandre Gramfort <notifications@github.com>
> wrote: In doc/source/whats_new.rst: > @@ -31,6 +31,9 @@ Changelog > > - Add
> new object :class:`mne.decoding.TimeDecoding` for decoding sensors' evoked
> response across time by `Jean-Remi King`_ > > + - Add support to append new
> channels from a list of Raw objects by `Chris Holdgraf`_ as you modified
> the mixin you added the feature to all objects (Epochs, Evoked etc...).
> Although only Raw is tested. — Reply to this email directly or view it on
> GitHub.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/mne-tools/mne-python/pull/2302/files#r34837076>.
>